test/powershell/Modules/Microsoft.PowerShell.Management/Get-Location.Tests.ps1

```diff
@@ -2,20 +2,80 @@
 # Licensed under the MIT License.
 Describe "Get-Location" -Tags "CI" {
     $currentDirectory=[System.IO.Directory]::GetCurrentDirectory()
+
     BeforeEach {
-	Push-Location $currentDirectory
+        Push-Location $currentDirectory
     }
 
     AfterEach {
-	Pop-location
+	    Pop-location
     }
 
     It "Should list the output of the current working directory" {
-
-	(Get-Location).Path | Should -BeExactly $currentDirectory
+	    (Get-Location).Path | Should -BeExactly $currentDirectory
     }
 
     It "Should do exactly the same thing as its alias" {
-	(pwd).Path | Should -BeExactly (Get-Location).Path
+	    (pwd).Path | Should -BeExactly (Get-Location).Path
+    }
+
+    It "Should throw an exception when missing an argument for parameter 'PSProvider'" {
+        { Get-Location -PSProvider } | Should -Throw "Missing an argument for parameter 'PSProvider'."
```

Comment:
We can not use localized strings. Please use `Should -Throw -ErrorId "..."` pattern.
Below too.